src/notation/internal/notationplayback.cpp

```diff
@@ -57,6 +57,7 @@ NotationPlayback::NotationPlayback(IGetScore* getScore,
 {
     notationChanged.onNotify(this, [this]() {
         updateLoopBoundaries();
+        m_playbackModel.reload();
```

Comment:
This works for the intent of fixing the referred bug, but not being familiar with the normal notification flow I wonder if it perhaps isn't overkill to reload the playbackModel on every notationChange?

Reply:
Normally, the playback model already updates after every command, based on info from the undo stack. The reason that this doesn't happen during score creation, is that the undo stack is locked (see NotationProject::loadTemplate), so the playback model gets no notifications from it.

So, we either need to make it so that the playback model is reloaded when the undo stack is unlocked, or we need to trigger a reload of the playback model manually after the score creation options have been applied.